scss/components/action-bar.scss

```diff
@@ -127,4 +127,8 @@
   &:first-child {
     padding-left: 0;
   }
+
+  &:hover {
+    filter: hue-rotate(240deg);
```

Comment:
I like this! Let's have it rotate to human-blue and add a transition so that it fades in.

Reply:
Getting close on the color (but still a little off and I'll try again tomorrow).

I pushed some code so transition can be checked out. I think because of hue-rotate, you sort of see the button transition from pink to green (very briefly) to blue. Or maybe I'm seeing things.
Also, check out dark mode (I think it works).

Two other ideas. Both are a little counter-intuitive because they dull the button on hover. But instead of hue, can filter on grayscale(100%) for gray image or invert (25%) for a duller pink. Both of these work with the transition.